### DISCUSSION TITLE
Display Projects (classic) by default if user has no Projects (beta)

### DISCUSSION THREAD

**Discussion by benlmyers** [community/community#19870] [Other Feature Feedback, Questions, & Ideas]
When in a repository, it can often be annoying to have to click "Projects", then "Projects (Classic)", then the project. It would be much faster if the "Projects (Classic)" category is displayed **only if the user has one or more Classic Projects and zero Beta Projects**.

Redundant click highlighted in red:

<img width="1259" alt="Screen Shot 2022-07-02 at 1 02 53 AM" src="https://user-images.githubusercontent.com/41796709/176987255-b8c53e10-7b65-4db8-a36a-f84b33bc68b2.png">

To address the potential counterpoint that Github wants users to create new Beta Projects, I think that if a button to *create a new Beta Project* was displayed next to the "New classic project" button (highlighted in red), the problem could just as easily be solved.

<img width="1256" alt="Screen Shot 2022-07-02 at 12 57 06 AM" src="https://user-images.githubusercontent.com/41796709/176987285-3392f132-c578-48c9-8426-1d54c775f407.png">

**Comment by Malvoz** [comment]
Same as:

- https://github.com/github-community/community/discussions/11193
- https://github.com/github-community/community/discussions/13386
